pkg/collectors/policyreport.go

```diff
@@ -95,15 +95,17 @@ func createPolicyReportListWatchWithClient(client dynamic.Interface, ns string)
 
 func getReports(clusterID string, pr *v1alpha2.PolicyReport) [][]string {
 	var metrics [][]string
+	var severity string
+
 	if clusterID == "" {
 		return metrics
 	}
-	category, policy, result, severity := "", "", "", ""
+
+	result := "fail"
+
 	for _, reportResult := range pr.Results {
 		var metric []string
-		if reportResult.Category != "" {
-			category = reportResult.Category
-		}
+
```

Comment:
I still think `severity` and `result` should be loop-scoped variables so that a previous result doesn't bleed over?
```suggestion
var severity string
result := "fail"

```

Reply:
Good point, will update.  Here's what the metric looks like now:
# HELP policyreport_info Open Cluster Management PolicyReport Info.
# TYPE policyreport_info gauge
policyreport_info{managed_cluster_id="a35dc54a-ea3e-48e8-b011-072511ea8f36",category="",policy="policy-test.policyreport-metric-noncompliant",result="fail",severity="low"} 1